doc/cephfs/cephfs-top.rst

```diff
@@ -72,11 +72,11 @@ By default, `cephfs-top` connects to cluster name `ceph`. To use a non-default c
 
   $ cephfs-top --cluster <cluster>
 
-`cephfs-top` refreshes stats every second by default. To chose a different refresh interval use::
+`cephfs-top` refreshes stats every second by default. To choose a different refresh interval use::
 
   $ cephfs-top -d <seconds>
 
-Interval should be greater or equal to 0.5 second. Fractional seconds are honoured.
```

Comment:
I do not understand what's wrong with having refresh interval >= 0.5 seconds?

Reply:
It should be >=0.5 seconds [here](https://github.com/ceph/ceph/blob/master/src/tools/cephfs/top/cephfs-top#L548) then. It's confusing whether 0.5 is inclusive or not. We could fix the error text instead.

Reply:
Yeah, fix the help text instead.